### PR TITLE
COL-536, move and expand steps to set up Apache config

### PR DESCRIPTION
**Pull request by johncrossman**
https://jira.ets.berkeley.edu/jira/browse/COL-536

Lastly, add clarity to instructions on Apache config and move to `docs/configuration.md`.

### CODE REVIEW

**Comment by johncrossman** [comment]
Fyi, you can see Markdown changes rendered at https://github.com/johncrossman/collabosphere/blob/COL-536_apache/docs/configuration.md

File: docs/configuration.md [hunk 1]
```diff
@@ -31,15 +63,13 @@ where
...
 
-## SuiteC Configuration Options
-
-SuiteC allows the following values to be configured:
+### Customizations
 
-### `app` configuration
+#### `app` configuration
 
 - `port`: The internal port on which the SuiteC app server should listen
```
Comment (paulkerschen) [inline docs/configuration.md]: @johncrossman Might be worth mentioning the `https` and `host` options here also. `host` should be set to `localhost` for local deployments.

File: docs/configuration.md [hunk 2]
```diff
@@ -31,15 +63,13 @@ where
 - `{hostname}` is tge server name, from the `$HOST` or `$HOSTNAME` environment variable or `os.hostname()`
```
Comment (paulkerschen) [inline docs/configuration.md]: @johncrossman tgypo

File: docs/configuration.md (hunk 1 above)
Comment (johncrossman) [inline docs/configuration.md]: I agree, @paulkerschen   Updatin'

**Comment by johncrossman** [comment]
Changes applied.  cc: @paulkerschen

**Pull request merged**